Sources/StreamChatUI/ChatMessageList/MessageComposer/ChatMessageComposerCommandCollectionViewCell.swift

```diff
@@ -39,11 +39,10 @@ open class _ChatMessageComposerCommandCellView<ExtraData: ExtraDataTypes>: View,
         backgroundColor = .clear
 
         commandNameLabel.font = uiConfig.font.bodyBold
+        commandNameLabel.textColor = uiConfig.colorPalette.text
 
         commandNameSubtitleLabel.font = uiConfig.font.body
         commandNameSubtitleLabel.textColor = uiConfig.colorPalette.subtitleText
-
-        commandNameLabel.textColor = uiConfig.colorPalette.text
     }
 
     override open func setUpLayout() {
```

Comment:
nit: can be oneliner aka `setUpLayout() {}`

Reply:
This is not empty btw 😄 You probably got misled by the Github's review 😛

Reply:
When I am looking at the method name I feel ashamed :D :D :D Thanks, this shouldn't really be empty in any case :D :D